packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1770,13 +1770,10 @@ export class AccountHandler {
     }
 
     const response = {
-      currentVersion: accountRecord.clientSalt ? 'v2' : 'v1',
-      clientSalt: accountRecord.clientSalt
-        ? accountRecord.clientSalt
-        : undefined,
-      upgradeNeeded:
-        !accountRecord.verifyHashVersion2 || !accountRecord.wrapWrapKbVersion2,
-    };
+      currentVersion: accountRecord.clientSalt ? 'v2' :'v1',
```

Comment:
Not sure if Prettier is prettier-ing these files, but probably a space after the `:`.
```suggestion
      currentVersion: accountRecord.clientSalt ? 'v2' : 'v1',
```